infrastructure/tooling/src/smoketest/checks/createClient.js

```diff
@@ -0,0 +1,32 @@
+const taskcluster = require('taskcluster-client');
+
+exports.scopeExpression = {
+  AllOf: [
+    "auth:create-client:project/taskcluster/smoketest/*",
+    "auth:reset-access-token:project/taskcluster/smoketest/*",
+    "project:taskcluster:smoketest:*",
+  ],
+};
+
+exports.tasks = [];
+
+exports.tasks.push({
+  title: 'Create client and use it (--target client)',
+  requires: ['ping-auth'],
+  provides: [
+    'target-client',
+  ],
+  run: async () => {
+    const auth = new taskcluster.Auth(taskcluster.fromEnvVars());
+    const randomId = taskcluster.slugid();
+    let clientId = `project/taskcluster/smoketest/${randomId}`;
+    const payload = {
+      "expires": taskcluster.fromNowJSON('1 hour'),
+      "description": `Create a client and use it ${clientId}`,
+      "scopes": [`auth:reset-access-token:project/taskcluster/smoketest/${randomId}`],
+    };
+    const createClient = await auth.createClient(clientId, payload);
+    const accessToken = createClient.accessToken;
+    console.log(accessToken);
```

Comment:
...so here, you'll need to make a new `taskcluster.Auth` object, but pass the updated credentials.  Something like
```js
const auth2 = new taskcluster.Auth({
  rootUrl: process.env.TASKCLUSTER_ROOT_URL,
  credentials: {clientId, accessToken},
});
```
and then use *that* object to call `auth2.resetAccessToken`.

Reply:
I will try it right away!

Reply:
Sorry for the delay. Lost my internet connectivity for a while. :)